psutil/_psutil_windows.c

```diff
@@ -1278,31 +1278,31 @@ psutil_proc_open_files(PyObject *self, PyObject *args) {
 
 
 /*
- Accept a filename's drive in native  format like "\Device\HarddiskVolume1\"
+ Accept a filename's drive in native format like "\Device\HarddiskVolume1\"
  and return the corresponding drive letter (e.g. "C:\\").
  If no match is found return an empty string.
 */
 static PyObject *
 psutil_win32_QueryDosDevice(PyObject *self, PyObject *args) {
-    LPCTSTR   lpDevicePath;
+    LPCWSTR   lpDevicePath;
     TCHAR d = TEXT('A');
     TCHAR     szBuff[5];
 
-    if (!PyArg_ParseTuple(args, "s", &lpDevicePath))
+    if (!PyArg_ParseTuple(args, "u", &lpDevicePath))
```

Comment:
Mmm.. I'm a bit worried about this. Did you try to run tests on both python 2 and 3 to make sure this doesn't break anything?

Reply:
It actually breaks few tests.
Do you think it is a good solution to use `u` on Python 2 et `s` on Python 3?